how-we-work/oncall-responsibilities.md

```diff
@@ -20,13 +20,13 @@ An on-call rotation begins on Monday at 9am PT. Each week, there is a primary an
 
 ##### Deployment summary table
 
-|                   | dev                            | staging                                                                                  | prod                                                                                     | dev-canary                                                |
-| ----------------- | ------------------------------ | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------------------------------------------- |
-| single-cell-infra | TFE auto                       | TFE auto                                                                                 | TFE manual                                                                               | TFE manual                                                |
-| data-portal       | Need to set up auto deployment | Github Action Auto Deploy                                                                | happy/script Deploy Manual                                                               | no deployment                                             |
-| explorer          | Github Action Auto Deploy      | single-cell-infra deploy script to kick off github action (also possible to run locally) | single-cell-infra deploy script to kick off github action (also possible to run locally) | Github Action Auto Deploy (does not auto rebase off main) |
+| repo                    | dev                            | staging                                                                                  | prod                                                                                     | dev-canary                                                |
+| ----------------------- | ------------------------------ | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| single-cell-infra       | TFE auto                       | TFE auto (excl. [explorer](https://github.com/chanzuckerberg/single-cell-infra/blob/a8d1a3cc5f36280de69f7250f4a6422a55d574fc/terraform/tfe/locals.tf.json#L21) infra)                                                                                 | TFE manual                                                                               | TFE manual                                                |
+| single-cell-data-portal | Need to set up auto deployment | Github Action Auto Deploy                                                                | happy/script Deploy Manual                                                               | no deployment                                             |
```

Comment:
Link an issue for "Need to set up auto deployment"?

Reply:
portal now auto-deploys to dev env, and now reflected in table.